src/app/components/banner/ValidationRules.js

```diff
@@ -0,0 +1,20 @@
+function validate(values = {}) {
+    const errors = {};
+    if (!values.title) {
+        errors.title = "Title is required!";
```

Comment:
Exclamation marks are not usually used in validation errors

Reply:
corrected, thanks for that. we don't want to be harsh :-)